src/components/ModalEye.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <section class="information-modal" v-show="modal">
+    <span v-on:click="close" class="close">X</span>
+    <nav class="navbar-modal">
+      <h4>Detalhes do usuário</h4>
+
+    </nav>
+
+    <div class="contain-all-information">
+      <div class="informations">
+        <div class="information-content">
+          <label>Cpf</label>
+          <span>{{ User_Prop.cpf }}</span>
+          <span></span>
+        </div>
+        <div class="information-content">
+          <label>Nome completo</label>
+          <span>{{ User_Prop.fullname }}</span>
+        </div>
+        <div class="information-content">
+          <label>Nascimento</label>
+          <span>{{ User_Prop.birthDate }}</span>
+        </div>
+      </div>
+
+      <div class="informations">
+        <div class="information-content">
+          <label>Celular</label>
+          <span>{{ User_Prop.phone }}</span>
+        </div>
+        <div class="information-content">
+          <label>Contato</label>
+          <span>
+            <p>Whatsapp</p>
+          </span>
+        </div>
+        <div class="information-content">
+          <label>Email</label>
+          <span class="modal-email">{{ User_Prop.email}}</span>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  props: {
+    User_Prop: {
+      type: Object
+    }
+  },
+
+  data() {
+    return {
+      modal: true
+    };
+  },
+  methods: {
+    close: function () {
+      if (this.modal == true) {
+        this.modal = false
+      }
+    },
```

Comment:
Deixe a função com um nome mais autoexplicativo,  o que close? Além disso não precisa de um this.modal == true, pode ser apenas this.modal
```suggestion
    closeModal() {
      if (this.modal) {
        this.modal = false
      }
    },
```